tsconfig/base.json

```diff
@@ -0,0 +1,39 @@
+{
+	"compilerOptions": {
+		// Target latest version of ECMAScript.
+		"target": "esnext",
+		// Search under node_modules for non-relative imports.
+		"moduleResolution": "node",
+		// Process & infer types from .js files.
+		"allowJs": true,
+		"jsx": "react",
+		// Don't emit; allow Babel to transform files.
+		"noEmit": true,
+		// Enable strictest settings like strictNullChecks & noImplicitAny.
+		"strict": true,
+		// Import non-ES modules as default imports.
+		"esModuleInterop": true,
+		"skipLibCheck": false,
+		"module": "esnext",
```

Comment:
Was `es2015`, now `esnext`. `esnext` seems to be required for `import()` syntax, which we're already using.